pages/common/cat.md

```diff
@@ -18,10 +18,10 @@
 
 `cat -n {{file}}`
 
-- Display "$" at end of each line.
+- Display "$" at end of each line:
 
 `cat -E {{file}}`
 
-- Display TAB characters as ^I.
```

Comment:
On MacOS X I get:
cat: illegal option -- E
cat: illegal option -- T

Those two options seem to work only for the linux cat command.

Reply:
Maybe we should have a macOS-specific `cat` page?

Reply:
Has this issue not come up before? There are other commands where options slightly differ depending on the OS (sed, ps and grep come to my mind). Best way IMO would be to restrict examples to those which works on all systems.

Reply:
We have platform specific folders. Any pages which are different in other platforms should have pages along with those options in those folders.

Reply:
Yeah, it has come up a time or 2 before. We usually solve it by copying said page to the platform-specific folder and adjusting that copy instead, since a client will always prefer a platform-specific version of a page over a `common/` one or one for another platform.